### PR TITLE
Milvus 2.4.11 Script, patch and Dockerfile

### PR DESCRIPTION
**Pull request by sumitd2**
## Checklist

- [X] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [X] Have you validated script on UBI 9 container
- [X] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [X] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: m/milvus/milvus-v2.4.11-ubi9.3.sh [hunk 1]
```diff
@@ -0,0 +1,203 @@
+#!/bin/bash -ex
+# ----------------------------------------------------------------------------
+#
+# Package       : milvus
+# Version       : 2.4.11
+# Source repo   : https://github.com/milvus-io/milvus
+# Tested on     : UBI 9.3 (docker)
```
Comment (seth-priya) [inline m/milvus/milvus-v2.4.11-ubi9.3.sh:7]: Lets remove the (docker) string for consistency

File: m/milvus/milvus-v2.4.11-ubi9.3.sh [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+  def package_info(self):
+    self.cpp_info.libs = tools.collect_libs(self)
+EOT
+}
+
+#Install centos and epel repos
+yum config-manager --add-repo https://mirror.stream.centos.org/9-stream/CRB/ppc64le/os
+yum config-manager --add-repo https://mirror.stream.centos.org/9-stream/AppStream//ppc64le/os
+yum config-manager --add-repo https://mirror.stream.centos.org/9-stream/BaseOS/ppc64le/os
+#rpm --import http://mirror.centos.org/centos/RPM-GPG-KEY-CentOS-Official
```
Comment (seth-priya) [inline m/milvus/milvus-v2.4.11-ubi9.3.sh:76]: can we delete if not needed?

File: m/milvus/milvus-v2.4.11.patch [hunk 3]
```diff
@@ -0,0 +1,435 @@
...
+-    image: minio/minio:RELEASE.2023-03-20T20-16-18Z
++    image: minio/minio:latest
+     ports:
+       - "9000:9000"
+       - "9001:9001"
+@@ -60,48 +56,6 @@ services:
+       timeout: 20s
+       retries: 3
+ 
+-  azurite:
```
Comment (seth-priya) [inline m/milvus/milvus-v2.4.11.patch:51]: what all is impacted by this?

**Review by seth-priya — COMMENTED** [review]
in general, for any dependencies being install from source, we need to start putting some additional error handling to ensure that the version being built (for example for python , golang etc.) is not already installed, this is to support some upcoming changes in currency

File: m/milvus/milvus-v2.4.11-ubi9.3.sh (hunk 1 above)
Comment (sumitd2) [reply m/milvus/milvus-v2.4.11-ubi9.3.sh:7]: We have tested it using docker, so its a good idea to inform.

File: m/milvus/milvus-v2.4.11.patch (hunk 3 above)
Comment (sumitd2) [reply m/milvus/milvus-v2.4.11.patch:51]: This is just for the golang tests.

File: m/milvus/milvus-v2.4.11-ubi9.3.sh (hunk 1 above)
Comment (seth-priya) [reply m/milvus/milvus-v2.4.11-ubi9.3.sh:7]: actually that's self implied when we say ubi 9.3

File: m/milvus/milvus-v2.4.11.patch (hunk 3 above)
Comment (seth-priya) [reply m/milvus/milvus-v2.4.11.patch:51]: noted

File: m/milvus/milvus-v2.4.11-ubi9.3.sh (hunk 1 above)
Comment (sumitd2) [reply m/milvus/milvus-v2.4.11-ubi9.3.sh:7]: I meant it wasnt podman.

Comment (seth-priya) [reply m/milvus/milvus-v2.4.11-ubi9.3.sh:7]: Okay, got it. Lets add  a comment in the script in that case, we don't want to complicate / make the text in these headers free form as currency / bulk search parses and uses the context from here.

**Comment by sumitd2** [comment]
> in general, for any dependencies being install from source, we need to start putting some additional error handling to ensure that the version being built (for example for python , golang etc.) is not already installed, this is to support some upcoming changes in currency

Okay, but in that case script will have to be tested in that kind of an environment, which would open a can of worms. As you know we only test in a fresh docker container to keep things simple enough.

**Comment by seth-priya** [comment]
> > in general, for any dependencies being install from source, we need to start putting some additional error handling to ensure that the version being built (for example for python , golang etc.) is not already installed, this is to support some upcoming changes in currency
> 
> Okay, but in that case script will have to be tested in that kind of an environment, which would open a can of worms. As you know we only test in a fresh docker container to keep things simple enough.

Yes, think of it as an extension built on top of the existing scripts and doing some pre/post processing.

**Review by seth-priya — APPROVED** [review]

**Pull request merged**